app/endpoints.js

```diff
@@ -18,6 +18,7 @@ const endpoints = {
   getServiceRecipientsFromOapi: options => `${organisationApiUrl}/api/v1/Organisations/${options.orgId}/service-recipients`,
   getSelectedServiceRecipientsFromOrdapi: options => `${orderApiUrl}/api/v1/orders/${options.orderId}/sections/service-recipients`,
   getSolutionsForSupplier: options => `${solutionsApiUrl}/api/v1/solutions?supplierId=${options.supplierId}`,
+  getSolution: options => `${solutionsApiUrl}/api/v1/solutions/${options.solutionId}`,
```

Comment:
can we reference bapi in this endpoint name?

Reply:
I agree. It's something I wanted to discuss with you guys about how do we tidy this up a little now.
I was thinking that the getEndpoint maybe should reference the API it is calling so it will look something like this;

`getEndpoint = ({ api, endpointLocator, options })`
So for this one it will be something like
`getEndpoint({ api: 'bapi', endpointLocator: 'getSolution', options })`

We can then maybe group the endpoints by APIs then too..

Something we can tidy up next